ws/src/server.rs

```diff
@@ -38,6 +38,9 @@ impl Server {
 		&self.addr
 	}
 
+	// Yes it's indeed too many -- but this is how our API goes
+	// TODO: Let's refactor this into a ServerOptions struct for the next major release
```

Comment:
Configuration object will just mean more code with no gains (the whole thing could be made private). We have a builder pattern implemented anyway.